ambari-server/src/main/python/ambari_server/setupSecurity.py

```diff
@@ -821,7 +821,7 @@ def setup_ldap(options):
 
   ssl_truststore_type_default = get_value_from_properties(properties, SSL_TRUSTSTORE_TYPE_PROPERTY, "jks")
   ssl_truststore_path_default = get_value_from_properties(properties, SSL_TRUSTSTORE_PATH_PROPERTY)
-  disable_endpoint_identification_default = get_value_from_properties(properties, LDAP_DISABLE_ENDPOINT_IDENTIFICATION, "False")
+  disable_endpoint_identification_default = get_value_from_properties(properties, LDAP_DISABLE_ENDPOINT_IDENTIFICATION, "True")
```

Comment:
This should remain false. By default endpoint identification should apply while the client is negotiating with the server during SSL handshake to make sure the client can trust the server.
The issue, why this change has been added previously, has been occurred only with OpenJDK (i.e. openjdk-1.8.0.181-3.b13.el7_5.x86_64). In this case customers can change this flag and disable endpoint identification (which is less secure but they trust the server they are connecting to)